File: pset6/mario.py
height = 0;
print("height:", end="")
while(True):
    height = int(input())
    if height >= 0 and height <= 23:
        break
    else:
        print("retry")

for i in range (height):
    for j in range (height-1-i):
        print(" ", end="")
    for j in range (i+1):
        print("#", end="")
    print("  ", end="")  #middle space
    for j in range(i+1):
        print("#", end="")
    print()
    

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height=0;
    printf("Height:");
    while(true)
    {
        height=get_int();
        if(height>=0 && height<=23)
        {
            break;
        }
        else
        {
            printf("retry:");
        }
    }
    for(int i=0;i<height;i++)
    {
        for(int j=0;j<height-1-i;j++) //loop for spaces before #s
        {
            printf(" ");
        }
        
        for(int j=0;j<=i;j++)   //loop to print first series of #s
        {
            printf("#");
        }
        
        printf("  ");    //middle space
        
        for(int j=0;j<=i;j++)
        {
            printf("#");
        }
        
        printf("\n");  //prompt to next line
    }
}